common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -88,14 +88,19 @@
     </script>
 </%def>
 
-<%def name="optional_include_mako(file, with_microsite=False)"><%
+<%def name="optional_include_mako(file, with_microsite=False, fallback=None, **kwargs)"><%
 # http://stackoverflow.com/q/21219531
 if with_microsite:
     file = microsite.get_template_path(file)
+tmpl = None
 try:
     tmpl = self.get_template(file)
 except TemplateLookupException:
-    pass
-else:
-    tmpl.render_context(context)
-%></%def>
+    if fallback:
+        try:
+            tmpl = self.get_template(fallback)
+        except TemplateLookupException:
+            pass
+if tmpl:
+    tmpl.render_context(context._locals(kwargs))
```

Comment:
Rather than adding template context using `**kwargs`, can we match the behavior of Mako's [`<%include>`](http://docs.makotemplates.org/en/latest/syntax.html#include) tag as much as possible? `<%include>` uses an `args` parameter, which takes key-value pairs. For example: `<%static:optional_include_mako file="template.html" args="foo='bar'">`